Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/01-Essentials/03-TestingYourFeatures/01-03-01-code-0001.swift

```diff
@@ -1,6 +1,8 @@
 import ComposableArchitecture
 import XCTest
 
+@testable import CounterApp
```

Comment:
Just mimicking what's going on in the Navigation part, where the main app target is imported